<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmArrangement2</class>
 <widget class="QMainWindow" name="frmArrangement2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arrangement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Create a new antenna  </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxAntennaType">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Array</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Points</string>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>541</width>
      <height>541</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Import array</string>
     </attribute>
     <widget class="QGroupBox" name="groupWorkplane">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Workplane</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Origin(m)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtOrigin">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDirection">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Normal Direction</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtRotatebyZ">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>190</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotate by Z(deg)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtAxisLength">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>230</y>
         <width>221</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Axis Length(m)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStartPick">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>25</y>
         <width>471</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Pick a face by Ctrl+Mouseleft</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtAngel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Angel-XY/YZ/XZ(deg)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkShowAxis">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>230</y>
         <width>71</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupArrange">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>521</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Antenna Arrange</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>X-distance(m)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Display Size(pixel)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDisplayPixel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Z-distance(m)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDistance_Z">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDistance_Y">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Y-distance(m)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDistance_X">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>100</y>
         <width>281</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtAntennaFile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>15</y>
         <width>221</width>
         <height>30</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Antenna </string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBrowse">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>15</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Input points</string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Display Size(pixel)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDisplayPixel_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>470</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Anntenna points</string>
      </property>
      <widget class="QPushButton" name="btnAddPoint">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Point</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRemovePoint">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>410</y>
         <width>221</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove Point</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tbPoints">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>481</width>
         <height>381</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
